### PR TITLE
Add schema fields to Sonicwall decoder

### PR DESCRIPTION
**Pull request by soynof**
|Related Issue|
|---|
| #16406|

## Description

With the aim of expanding the coverage of the decoders, this PR adds the schema fields to Sonicwall decoder.

### CODE REVIEW

**Review by cborla — APPROVED** [review]
lgtm

**Review by JorgeMarinoDev — APPROVED** [review]
LGTM

**Review by nmkoremblum — CHANGES_REQUESTED** [review]
I am still receiving the following error:
```
Failed to read entry /root/repos/engine/src/engine/ruleset/decoders/sonicwall/sonicwall-firewall-generated.yml: An error occurred while trying to validate "decoder/sonicwall-firewall-generated/0": Parse stage: Building asset "decoder/sonicwall-firewall-generated/0" failed: Error building parser "logpar": An error occurred while parsing a log: Field 'observer.egress.interface.name' not found in schema
```

File: src/engine/ruleset/environments/wazuh-environment.yml [hunk 1]
```diff
@@ -69,4 +69,4 @@ decoders:
   - decoder/syslog/0
 
 outputs:
-  - output/file-output/0
+  - output/file-output/0
```
Comment (nmkoremblum) [inline src/engine/ruleset/environments/wazuh-environment.yml:72]: Please fix the newline at the end

**Review by nmkoremblum — CHANGES_REQUESTED** [review]

**Review by JcabreraC — APPROVED** [review]
LGTM !

**Pull request merged**